### DISCUSSION TITLE
UI based alerting rule management.

### DISCUSSION THREAD

**Discussion by weixiang1862** [apache/skywalking#11128] [Ideas]
I saw ui-template was categoried as management data in skywalking. Is it possible to implement alerting rule management service too? Based on this, alerting rule can be management in ui.

The alerting rule managed in storage can coexist with that in configuration, and it wont breaking the previous using way.

**Comment by wu-sheng** [comment]
No, there isn't. Upstream doesn't have management feature UI, including alerting, user login, authentication...

The maintainers don't have intetests to work these, as there is not a unique way to do so.

**Reply by weixiang1862** [reply]
Ok, thanks.
